aspnetcore/mvc/controllers/areas.md

```diff
@@ -185,13 +185,4 @@ Browsing to *http://<yourApp>/products*, the `Index` action method of the `HomeC
 
 ## Publishing Areas
 
-To publish all views of the areas folder, in the `project.json` file include an entry in the `publishOptions`'s `include` node like below:
-
-```text
-"publishOptions": {
-     "include": [
-       "Areas/**/*.cshtml",
-       ....
-       ....
-     ]
-   ```
+All `*.cshtml` and `wwwroot/**` files are published to output when `<Project Sdk="Microsoft.NET.Sdk.Web">` is included in the *.csproj* file.
```

Comment:
If that's something that isn't automatically added by all of the project templates, it might be worth showing an example.